src/blockstore/transaction_status.zig

```diff
@@ -0,0 +1,245 @@
+    UnsupportedVersion,
+
+    /// Transaction loads a writable account that cannot be written
+    // #[error("Transaction loads a writable account that cannot be written")]
+    InvalidWritableAccount,
+
+    /// Transaction would exceed max account limit within the block
+    // #[error("Transaction would exceed max account limit within the block")]
+    WouldExceedMaxAccountCostLimit,
+
+    /// Transaction would exceed account data limit within the block
+    // #[error("Transaction would exceed account data limit within the block")]
+    WouldExceedAccountDataBlockLimit,
+
+    /// Transaction locked too many accounts
+    // #[error("Transaction locked too many accounts")]
+    TooManyAccountLocks,
+
+    /// Address lookup table not found
+    // #[error("Transaction loads an address table account that doesn't exist")]
+    AddressLookupTableNotFound,
+
+    /// Attempted to lookup addresses from an account owned by the wrong program
+    // #[error("Transaction loads an address table account with an invalid owner")]
+    InvalidAddressLookupTableOwner,
+
+    /// Attempted to lookup addresses from an invalid account
+    // #[error("Transaction loads an address table account with invalid data")]
+    InvalidAddressLookupTableData,
+
+    /// Address table lookup uses an invalid index
+    // #[error("Transaction address table lookup uses an invalid index")]
+    InvalidAddressLookupTableIndex,
+
+    /// Transaction leaves an account with a lower balance than rent-exempt minimum
+    // #[error("Transaction leaves an account with a lower balance than rent-exempt minimum")]
+    InvalidRentPayingAccount,
+
+    /// Transaction would exceed max Vote Cost Limit
+    // #[error("Transaction would exceed max Vote Cost Limit")]
+    WouldExceedMaxVoteCostLimit,
+
+    /// Transaction would exceed total account data limit
+    // #[error("Transaction would exceed total account data limit")]
+    WouldExceedAccountDataTotalLimit,
+
+    /// Transaction contains a duplicate instruction that is not allowed
+    // #[error("Transaction contains a duplicate instruction ({0}) that is not allowed")]
+    DuplicateInstruction, //(u8),
+
+    /// Transaction results in an account with insufficient funds for rent
+    // #[error("Transaction results in an account ({account_index}) with insufficient funds for rent")]
+    InsufficientFundsForRent, // { account_index: u8 },
+
+    /// Transaction exceeded max loaded accounts data size cap
+    // #[error("Transaction exceeded max loaded accounts data size cap")]
+    MaxLoadedAccountsDataSizeExceeded,
+
+    /// LoadedAccountsDataSizeLimit set for transaction must be greater than 0.
+    // #[error("LoadedAccountsDataSizeLimit set for transaction must be greater than 0.")]
+    InvalidLoadedAccountsDataSizeLimit,
+
+    /// Sanitized transaction differed before/after feature activiation. Needs to be resanitized.
+    // #[error("ResanitizationNeeded")]
```

Comment:
can we remove these \#\[errors\]

Reply:
4d870529c9c26d3dd546c97032bd7f6dde057bb4